src/main/kotlin/no/nav/tilleggsstonader/sak/utbetaling/tilkjentytelse/TilkjentYtelseService.kt

```diff
@@ -58,17 +59,21 @@ class TilkjentYtelseService(
      * Legger til en nullandel ved første iverksetting for en behandling, dersom det ikke finnes andeler som skal iverksettes
      * for forrige måned. Dette er for å kunne sjekke status på iverksetting uten utbetalinger.
      *
+     * Nullandels fom settes til måneden det minste av måneden det iverksettes for eller måneden før første andelen dersom en andel
+     * Eks hvis man i januar innvilger noe for jan-mai som ikke har sats, då legges en nullandel for desember inn.
+     *
```

Comment:
Denne kommentaren skjønte jeg ikke. Ufullstendige setninger og hva er behovet for endringen her?

Reply:
Behover er beskrevet på rad 2 med et eksempel
https://github.com/navikt/tilleggsstonader-sak/pull/517/commits/d1b0038889362edc82baebd77d42f89679666a74